AvailabilityCartConnector/7.3.0/installer-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "wire-plugin": [
```

Comment:
also should be unwire part. I guess you need to add this manually. You can check existing manifests